breeze

```diff
@@ -290,7 +290,7 @@ Usage: ${CMDNAME} [COMMAND] [FLAGS]  -- <EXTRA_ARGS>
 
 The swiss-knife-army tool for Airflow testings. It allows to perform various test tasks:
 
-  * Enter interactive environment when no command flags are specified (default behaviour)
+  * Run interactive terminal when no command flags are specified (default behaviour)
```

Comment:
IT's not terminal. It's the whole environment using integrations, runtime, kubernetes, databases etc. I still think environment is better name,